src/router.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3.7
+
+import socket
+import binascii
+import struct
+from pascy.l2 import *
+from pascy.l3 import *
+from pascy.l4 import *
+from pascy.layer import *
+from pascy.fields import *
+from dataclasses import dataclass
+from collections import namedtuple
+
+
+@dataclass
+class RoutingTable:
+    """
+    Holds the routing table the a router is using. Enables 
+    to resolve the next interface to forward a packet. """
+
+    ENDIANITY = 'big'
+
+    routing_table: list
+    
+    def route(self, dest_ip):
+        '''
+        Determine the interface through which the router should 
+        forward packets to get to the destination ip address.
+
+        :param dest_ip: the ip address of the destination
+        :returns: the resolved interface ip
+        :rtype: string
+        '''
+        self.routing_table.sort(reverse=True, key= lambda entry : entry[1])
+        for entry in self.routing_table:
+            dest_ip_int = int.from_bytes(socket.inet_aton(dest_ip), RoutingTable.ENDIANITY)
```

Comment:
For the future, you could use the `ipaddress` module, it could help you manage, build, and parse IP addresses and subnets.
This helps avoid using bits operations